third_party/libusb/naclport/src/libusb_over_chrome_usb.cc

```diff
@@ -523,6 +535,12 @@ int LibusbOverChromeUsb::LibusbOpen(
   if (!result.is_successful()) {
     GOOGLE_SMART_CARD_LOG_WARNING << "LibusbOverChromeUsb::LibusbOpen " <<
         "request failed: " << result.error_message();
+    // Modify the devices (fake) bus number that we report so that PCSC will 
+    // retry to connect to the device once it updates the device list.
+    uint32_t newBusNumber = LibusbGetBusNumber(dev) + 1;
```

Comment:
nit: Let's do the explicit cast:

`   static_cast<uint32_t>(LibusbGetBusNumber(dev))`

, since relying on the implicit integral promotion can be tricky to notice.

Reply:
Done